CSC8503/LevelManager.cpp

```diff
@@ -68,7 +68,7 @@ LevelManager::LevelManager() {
 
 	//preLoadtexID   I used Guard mesh to player and used rigMesh to guard   @(0v0)@  Chris 12/02/1998
 	mAnimation->PreloadMatTextures(*mRenderer, *mMeshes["Rig"], *mMaterials["Rig"], mGuardTextures);
-	mAnimation->PreloadMatTextures(*mRenderer, *mMeshes["Guard"], *mMaterials["Guard"], mPlayerTextures);
+	mAnimation->PreloadMatTextures(*mRenderer, *mMeshes["Guard"], *mMaterials["Guard_Red"], mPlayerTextures);
```

Comment:
So I just changed how the whole material system worked with my most recent PR, so you may need to go through and change how this is loaded (now being done in the AddPlayerToWorld function)

Reply:
I noticed the change in the material module, it seems that just preloadMaterial is enough. 
Tomorrow I'll try to work with Eren on how to properly use the new material system and remove the redundancies!